contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -569,13 +577,24 @@ function Start-OpenSSHBuild
     {
         $cmdMsg += "/noconlog"
     }
-    
-    $msbuildCmd = Get-VS2017BuildToolPath
-    if($msbuildCmd -eq $null)
+
+    if ($msbuildCmd = Get-VS2019BuildToolPath)
```

Comment:
```suggestion
    if ($msbuildCmd -eq Get-VS2019BuildToolPath)
```

Reply:
This isn't equivalency, but assignment + condition, i.e. `$msbuildCmd` gets the return of `Get-<Whatever>` and then the conditional checks if `$msbuildCmd` is falsey.